docker-compose-dev.yml

```
@@ -0,0 +1,25 @@
+version: '2'
+services:
+  db:
+    image: "postgres:12.2"
+    restart: always
+    env_file:
+      - src/.env
+    ports:
+      - '5432:5432'
+    volumes:
+      - /usr/local/var/postgres:/var/lib/postgresql/data
```

would suggest to keep it local and add it to `.gitignore`

```
./postgres_data:/var/lib/postgresql/data
```

---

why ?

---

[R] lets use a named volume here, writing to a directory assumes this path isn't occupied on the machine by another setup and will just bloat the desk, using a volume will let you clean the unused volumes with `docker purge` when needed.
```suggestion
      - rubyfunctions_db:/var/lib/postgresql/data
```